File: BasicInputOutput/C++/AreaPerimeterCircle.cpp
#include <iostream>
using namespace std;

int main() {
    float pi = 3.14f, radius;
    cout << "Enter the radius of the circle: ";
    cin >> radius;
    cout << "Area of Circle: " << (pi * radius * radius) << endl;
    cout << "Perimeter of Circle: " << (2 * pi * radius) << endl;
    return 0;
}

File: BasicInputOutput/Java/AreaPerimeterCircle.java
import java.util.Scanner;

class AreaPerimeterCircle{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        float pi = 3.14f;
        System.out.println("Enter Radius of Circle : ");
        float radius = sc.nextFloat();

        System.out.println("Area of Circle: "+(pi*radius*radius)); // pi * r^2
        System.out.println("Perimeter of Circle: "+(2*pi*radius)); // 2 * pi * r
    }
}